src/SFML/Graphics/GLExtensions.hpp

```diff
@@ -41,28 +41,43 @@
 // All functionality beyond that is optional
 // and has to be checked for prior to use
 
+namespace sf::priv
+{
+// Stand-ins for desktop OpenGL feature-check flags
+// NOLINTBEGIN(readability-identifier-naming)
+extern int SF_GL_OES_multitexture;
```

Comment:
Let’s use C++17 `inline` variables to avoid the need for `extern` plus a definition in a .cpp file.